DEVELOPMENT.md

```diff
@@ -51,30 +57,30 @@
 
 - Update all unit test golden files:
 
-  ```sh
-  mvn test -DupdateUnitGoldens
-  ```
+    ```sh
+    mvn test -DupdateUnitGoldens
+    ```
 
 - Update a single unit test golden file, for example `JavaCodeGeneratorTest.java`:
 
-  ```sh
-  mvn test -DupdateUnitGoldens -Dtest=JavaCodeGeneratorTest
-  ```
+    ```sh
+    mvn test -DupdateUnitGoldens -Dtest=JavaCodeGeneratorTest
+    ```
 
 - Run a single integration test for API like `Redis`, it generates Java source
   code using the Java microgenerator and compares them with the goldens files
   in `test/integration/goldens/redis`.
 
-  ```sh
-  bazel test //test/integration:redis
-  ```
+    ```sh
+    bazel test //test/integration:redis
```

Comment:
Should all of these `bazel` commands be using `bazelisk` instead? (CC @suztomo )

Reply:
Yes, use bazelisk where applicable.